### DISCUSSION TITLE
[Vue] Event for hits loaded

### DISCUSSION THREAD

**Discussion by LefanTan** [algolia/instantsearch#6188] [Q&A]
Hi folks, we're looking for a good way to detect when the hits are fully loaded. Currently, we have a custom component listening to `this.state.hits` in a custom component connecting to `connectHits`, but this isn't entirely accurate since we notice that the hits changes during a query change.

**Comment by Haroenv** [comment]
What is this for? Do you want to display a loader before the first hits get displayed?

**Reply by LefanTan** [reply]
We want to make an API call based on the hits retrieved by algolia

**Reply by Haroenv** [reply]
I would recommend that you do this inside a proxy of the search client, after the request has returned, a bit like this: https://www.algolia.com/doc/guides/building-search-ui/going-further/conditional-requests/js/
